packages/cli-plugin/examples/dx-parcel-minifier/src/lib.rs

```diff
@@ -0,0 +1,98 @@
+use std::fs;
+
+use dioxus_cli_plugin::*;
+use exports::plugins::main::definitions::Guest;
+use plugins::main::{
+    imports::{get_project_info, log},
+    toml::{Toml, TomlValue},
+    types::{CommandEvent, PluginInfo, ResponseEvent, RuntimeEvent},
+};
+
+struct CSSMinifer;
+
+fn minify_css() -> Result<ResponseEvent, ()> {
+    for entry in fs::read_dir("/dist").unwrap() {
```

Comment:
This could be different if the user changes the output directory

Reply:
That part isn't completely finished, but the "/dist" directory would be sandboxed for all the plugins and would be just set to whatever the host project has in the asset directory and output directory configs, so any plugin would just read and write from "/dist" and "/assets" and it would pull from where the configs are set to. At the moment there is a `has_output_dir` and `has_asset_dir` variables in the `get_project_info` function that tells the plugin if the host project has these defined, which would guarantee the existence of the "asset" and "dist" directories to be available to plugins since the plugin sandboxes these when they are loaded [here](https://github.com/ealmloff/dioxus/blob/b6e83d03dc49ad75f58033542b5a031dd23b0074/packages/cli/src/plugin/mod.rs#L291), but maybe a `get_asset_dir -> option<path>` function would be better?

Reply:
We also just create the dist directory [here](https://github.com/ealmloff/dioxus/blob/b6e83d03dc49ad75f58033542b5a031dd23b0074/packages/cli/src/cli/build.rs#L73C24-L73C24) anyway, so just defaulting to "/dist" if it's not available in the config shouldn't be too much of an issue

Reply:
Oh, interesting! I think `get_asset_dir` would be a bit more clear, but mapping the /dist and /assets directory also works